tests/view_tests/tests/test_debug.py

```diff
@@ -273,6 +273,12 @@ def test_template_not_found_error(self):
 
 class ExceptionReporterTests(SimpleTestCase):
     rf = RequestFactory()
+    DOGECOIN_SETUP = {'some-key': 'DOGECOIN_SETUP-nicekey',
```

Comment:
Could you remove the `DOGECOIN` references? Also, I don't think this needs to be a module constant. It probably doesn't need to check 6 values either.